201-blob-storage-private-endpoint/azuredeploy.json

```diff
@@ -375,8 +379,12 @@
       "type": "Microsoft.Compute/virtualMachines",
       "name": "[parameters('vmName')]",
       "location": "[parameters('location')]",
+      "identity": {
+        "type": "SystemAssigned"
```

Comment:
is this used?  I don't see a role assigned anywhere...

Reply:
I copied and pasted the ARM template from the equivalent sample on my GitHub account which makes use of a system-assigned managed identity to run a more complex Bash script. I removed the system-assigned managed identity from the ARM template here in Azure Quickstart Templates. :)